mock/src/components/Commands.ts

```diff
@@ -0,0 +1,106 @@
+import { REPLFunction } from "./REPLFunction";
+import { Dispatch, SetStateAction, useState } from "react";
+import { mockedDataMap, mockedResultsMap } from "./mockedJson";
+import { HistoryEntry } from "./REPL";
+
+// An export that REPLInput can use to map certain commands to functions
+export const commandMap = new Map<string, REPLFunction>();
+
+// Create a varaible to store the file
+let loadedData: string[][];
+let file: { data: string[][]; hasHeaders: boolean } | undefined;
+
+export function addCommand(
+  commandName: string,
+  commandFunction: REPLFunction
+): void {
+  commandMap.set(commandName, commandFunction);
+}
+
+export function removeCommand(commandName: string): void {
```

Comment:
This function is a great feature for allowing developers to remove a command and makes it easier to alter code